modules/core/frontend/lib/features/entities/mutations.ts

```diff
@@ -63,11 +57,12 @@ export const useCreateEntityMutation = <T extends EntityProperties>(
   return useMutation({
     mutationKey: ["createEntity", entityPath],
     mutationFn: async (entityData: Partial<T>) => {
-      upsert("Creating records...", {
+      const toastId = Math.random().toString(36).slice(2, 15);
+      toast("Creating records...", {
```

Comment:
```suggestion
      const toastId = toast("Creating records...", {
```

`toast` returns the id of the created toast, you don't need to create one yourself